# Visualization Data Message
std_msgs/Header header

# Data in JSON format for 3D visualization
string json_data
